### PR TITLE
Added EU status page link

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by mcocaro — APPROVED** [review]
LGTM

File: modules/ROOT/pages/maintenance-and-upgrade-policy.adoc [hunk 1]
```diff
@@ -13,6 +13,12 @@ All maintenance to the Runtime Manager console, platform services, and worker cl
 
 MuleSoft's policy is to perform all maintenance transparently and without any downtime to your integration(s) running in the worker cloud wherever possible. In cases where downtime for the console, platform services, or workers is unavoidable, MuleSoft communicates these maintenance windows at least eight hours in advance and typically seven days in advance, with the exception of critical security patches which may be fixed immediately.
```
Comment (kbostrom) [inline modules/ROOT/pages/maintenance-and-upgrade-policy.adoc]:
```suggestion
MuleSoft policy is to perform all maintenance transparently and without any downtime to your integrations running in the worker cloud wherever possible. In cases where downtime for the console, platform services, or workers is unavoidable, MuleSoft communicates these maintenance windows at least eight hours in advance and typically seven days in advance, with the exception of critical security patches, which might be fixed immediately.
```

File: modules/ROOT/pages/maintenance-and-upgrade-policy.adoc [hunk 2]
```diff
@@ -13,6 +13,12 @@ All maintenance to the Runtime Manager console, platform services, and worker cl
...
 
-View the live status and detailed service history for the Runtime Manager console, CloudHub platform services, and the CloudHub worker cloud on http://status.mulesoft.com/[status.mulesoft.com].
+View the live status and detailed service history for the Runtime Manager console, CloudHub platform services, and the CloudHub worker cloud:
```
Comment (kbostrom) [inline modules/ROOT/pages/maintenance-and-upgrade-policy.adoc]:
```suggestion
View the current status and detailed service history for the Runtime Manager console, CloudHub platform services, and the CloudHub worker cloud:
```

**Review by kbostrom — APPROVED** [review]
Here you go!

**Pull request merged**